editor/editor_file_dialog.h

```diff
@@ -145,8 +145,8 @@ class EditorFileDialog : public ConfirmationDialog {
 	bool show_hidden_files;
```

Comment:
This one too.

Reply:
(Might be worth doing a second pass grepping for `bool [a-zA-Z0-9_]+;` in headers I guess. Could be a follow up PR if there are many.)